provider/cmd/pulumi-resource-aws-static-website/website.ts

```diff
@@ -129,9 +129,20 @@ export class Website extends pulumi.ComponentResource {
     }
 
     private getOrganizationName(): string {
-        const [ organization ] = execSync(`pulumi --stack='${pulumi.getStack()}' stack`)
+        const config = new pulumi.Config();
+        const organizationNameOverride = config.get("pulumiOrganizationName");
+        if (organizationNameOverride && organizationNameOverride != "") {
+            return organizationNameOverride;
+        }
+
+        const [ organization = "" ] = execSync(`pulumi --stack=${pulumi.getStack()} stack`)
             .toString()
-            .match(/(?<=Owner: )[^\n]+/)!;
+            .match(/(?<=Owner: )[^\n]+/) ?? [];
+
+        if (organization == "") {
+            return execSync("pulumi whoami").toString();
+        }
+
```

Comment:
should we log some type of error message here if we still cannot find the org name by saying something like "set the organization name in your config file to resolve this error by running `pulumi config set pulumiOrganizationName <org name>", so the user knows about the escape hatch?

Reply:
Yeah good point, I'll add an error for this case.